### ISSUE TITLE
怎么在中断训练后恢复训练？

### ISSUE THREAD

**Issue by Kevinlaw95** [roboterax/humanoid-gym#59]
我训练了一段时间后，使用ctrl+c终止当前终端的训练脚本，怎么重新恢复训练？
我用了命令python scripts/train.py --task=humanoid_ppo --run_name v1 --resume --load_run Apr03_16-53-42_v1 --checkpoint 1800  --headless --num_envs 4096
但是训练次数依然是从0开始的，我的Apr06_14-10-30_v1目录下确认有model_1800.pt。
日志如下：
python scripts/train.py --task=humanoid_ppo --run_name v2 --resume --load_run Apr06_14-10-30_v1 --checkpoint 1800  --headless --num_envs 4096
Importing module 'gym_38' (/home/ecs-user/Desktop/isaacgym/python/isaacgym/_bindings/linux-x86_64/gym_38.so)
Setting GYM_USD_PLUG_INFO_PATH to /home/ecs-user/Desktop/isaacgym/python/isaacgym/_bindings/linux-x86_64/usd/plugInfo.json
PyTorch version 2.4.0
Device count 1
/home/ecs-user/Desktop/isaacgym/python/isaacgym/_bindings/src/gymtorch
Using /home/ecs-user/.cache/torch_extensions/py38_cu124 as PyTorch extensions root...
Emitting ninja build file /home/ecs-user/.cache/torch_extensions/py38_cu124/gymtorch/build.ninja...
Building extension module gymtorch...
Allowing ninja to set a default number of workers... (overridable by setting the environment variable MAX_JOBS=N)
ninja: no work to do.
Loading extension module gymtorch...
Setting seed: 5
Not connected to PVD
+++ Using GPU PhysX
Physics Engine: PhysX
Physics Device: cuda:0
GPU Pipeline: enabled
/home/ecs-user/miniconda3/envs/humanoid_gym/lib/python3.8/site-packages/torch/functional.py:513: UserWarning: torch.meshgrid: in an upcoming release, it will be required to pass the indexing argument. (Triggered internally at /opt/conda/conda-bld/pytorch_1720538438750/work/aten/src/ATen/native/TensorShape.cpp:3609.)
  return _VF.meshgrid(tensors, **kwargs)  # type: ignore[attr-defined]
Actor MLP: Sequential(
  (0): Linear(in_features=705, out_features=512, bias=True)
  (1): ELU(alpha=1.0)
  (2): Linear(in_features=512, out_features=256, bias=True)
  (3): ELU(alpha=1.0)
  (4): Linear(in_features=256, out_features=128, bias=True)
  (5): ELU(alpha=1.0)
  (6): Linear(in_features=128, out_features=12, bias=True)
)
Critic MLP: Sequential(
  (0): Linear(in_features=219, out_features=768, bias=True)
  (1): ELU(alpha=1.0)
  (2): Linear(in_features=768, out_features=256, bias=True)
  (3): ELU(alpha=1.0)
  (4): Linear(in_features=256, out_features=128, bias=True)
  (5): ELU(alpha=1.0)
  (6): Linear(in_features=128, out_features=1, bias=True)
)
WARNING - Could not sort runs by month: time data 'exp' does not match format '%b'
Loading model from: /home/ecs-user/Desktop/humanoid-gym/logs/XBot_ppo/Apr06_14-10-30_v1/model_1800.pt
/home/ecs-user/Desktop/humanoid-gym/humanoid/algo/ppo/on_policy_runner.py:290: FutureWarning: You are using `torch.load` with `weights_only=False` (the current default value), which uses the default pickle module implicitly. It is possible to construct malicious pickle data which will execute arbitrary code during unpickling (See https://github.com/pytorch/pytorch/blob/main/SECURITY.md#untrusted-models for more details). In a future release, the default value for `weights_only` will be flipped to `True`. This limits the functions that could be executed during unpickling. Arbitrary objects will no longer be allowed to be loaded via this mode unless they are explicitly allowlisted by the user via `torch.serialization.add_safe_globals`. We recommend you start setting `weights_only=True` for any use case where you don't have full control of the loaded file. Please open an issue on GitHub for any issues related to this experimental feature.
  loaded_dict = torch.load(path)

**Comment by sunmengmm** [comment]
请问是如何解决的呢
